public/src/bc5SupportDev/java/com/revenuecat/purchases/models/BC5StoreProduct.kt

```diff
@@ -0,0 +1,28 @@
+package com.revenuecat.purchases.models
+
+import android.os.Parcelable
+import com.android.billingclient.api.ProductDetails
+import com.android.billingclient.api.SkuDetails
+import com.revenuecat.purchases.ProductType
+import com.revenuecat.purchases.models.ComparableData
+import com.revenuecat.purchases.models.StoreProduct
+import com.revenuecat.purchases.parceler.JSONObjectParceler
+import kotlinx.parcelize.Parcelize
+import kotlinx.parcelize.RawValue
+import kotlinx.parcelize.TypeParceler
+import org.json.JSONObject
+
+@Parcelize
+@TypeParceler<JSONObject, JSONObjectParceler>()
+private data class BC5StoreProduct(
```

Comment:
From Maddie: cesar brought up that we should definitely have a better name for this if it's going to be public